server/src/db/generate_random_users.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import random
+from math import sqrt, pi, cos, sin
+def rstring(length: int) -> str:
+  chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890'
+  return ''.join([random.choice(chars) for _ in range(length)])
+
+def main():
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--number', type=int, required=True)
+  parser.add_argument('--latitude', type=float, default=34.0156128)
+  parser.add_argument('--longitude', type=float, default=-118.5030432)
+  parser.add_argument('--radius', type=float, required=True)
+  parser.add_argument('--baseid', type=int, default=999)
+  parser.add_argument('--minart', type=int, default=0)
+  parser.add_argument('--maxart', type=int, default=5)
+  parser.add_argument('--artpercent', type=float, default=0.35)
+  args = parser.parse_args()
+  print("INSERT INTO user (id, email, username, password) VALUES ")
+  print(',\n'.join(["({}, '{}', 'test_{}', '{}')".format(args.baseid + i,
+                                                        rstring(20)+"@gmail.com",
+                                                        rstring(6),
+                                                        'password') for i in range(args.number)]))
+  print(';')
+  art = []
+  for i in range(args.baseid, args.baseid + args.number):
+    if random.random() < args.artpercent:
+      for k in range(random.randint(args.minart, args.maxart + 1)):
+        r = args.radius * sqrt(random.random())
+        theta = random.random() * 2 * pi
+        art.append("('test_art', '{}', '{}', 'https://wanderlust-images.s3-us-west-2.amazonaws.com/f87d1363-cfad-4270-8f17-30c2fe671306', 1, {})".format(
+                    args.latitude + r * cos(theta),
+                    args.longitude + r * sin(theta),
+                    i))
```

Comment:
I may be misinformed, but I think there is an unimportant mistake here that also exists in other portions of the app (that I wrote... womp). I have been informed that the earth is a "sphere" and that this does not "work the way I want it to". However, this should be pretty much a circle in most locations, so I'm not concerned 👍.

Also, I think this is the second kind of this Python script in the repo. If we cared about long term maintenance/cleanup, I would suggest you remove the other one since I like this one more.